src/main/java/com/fullcontact/api/libs/fullcontact4j/http/FCRequest.java

```diff
@@ -36,6 +36,10 @@ public String getParam(String key) {
      */
     protected abstract void makeRequest(FullContactApi api, Callback<T> callback);
 
+    protected T makeRequest2(FullContactApi api) {
+        return null;
```

Comment:
The specific requests all implement `makeRequest`. I didn't want to have to change all of those classes, and now the card reader request is the only one using `makeRequest2`, so only that class Overrides it,